Formula/libao.rb

```diff
@@ -13,11 +14,30 @@ class Libao < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "automake" => :build
```

Comment:
Presuming these are only required for the `head` you need to put them in a block so not everyone has to install them to build the formula:

```ruby
head do
  url "https://git.xiph.org/libao.git"
  depends_on "automake" => :build
  depends_on "autoconf" => :build
  depends_on "libtool" => :build
end
```